syntaxes/prql.tmLanguage.json

```diff
@@ -82,20 +138,55 @@
         {
           "name": "keyword.operator.new",
           "match": "\\{[^}]*}"
+        },
+        {
+          "include": "#escape"
         }
       ]
     },
-    "strings": {
+    "unicode-bidi": {
+      "name": "invalid.illegal",
+      "match": "(\u202A|\u202B|\u202D|\u202E|\u2066|\u2067|\u2068|\u202C|\u2069)"
```

Comment:
Out of interest, what are these?

Reply:
This marks the characters with a red color if it comes across any of the Unicode bidirectional chars.

These are from chapter "2.1 Compiler Security" on page 2 of the [Trojan Source whitepaper](https://trojansource.codes/trojan-source.pdf).

Reply:
It looks like VS Code handles these already. Is there a reason they should be here? We should only have things that add value here, because more things makes it more difficult to improve and maintain...